### PR TITLE
Setup documentation

### PR DESCRIPTION
**Pull request by robertkbk**

### CODE REVIEW

**Review by t0ms0n00 — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,8 +1,64 @@
...
+- Node.js
+
+## Setting up
+
+1. Clone this repository and navigate to root directory.
+   ```
+   git clone https://github.com/war-in/suu-nft-project
+   ```
+
+1. Build and run Docker containers in a compose file:
```
Comment (war-in) [inline README.md:24]:
```suggestion
2. Build and run Docker containers in a compose file:
```

File: README.md [hunk 2]
```diff
@@ -1,8 +1,64 @@
...
+1. In a separate terminal, navigate to the `<root>/backend` directory and run a [Truffle](https://trufflesuite.com/docs/truffle/) node.
+   ```
+   npx truffle console
+   ```
+   This command will run a Truffle instance in console mode.
+   When in Truffle console, run `migrate` command to build and deploy smart contracts to Ganache node.
+   Whenever You make any changes to smart contracts, run a migration.
+   The output of the command should look simillar to:
+   ```
+   truffle(development)> migrate
```
Comment (war-in) [inline README.md:56]: I'm not sure, but we probably found that `migrate` is not enough. There was some `force migrate` command

File: README.md (hunk 1 above)
Comment (robertkbk) [reply README.md:24]: Markdown autonumbers the ordered list when enumerated with `1.` (You can see on the branch).

File: README.md (hunk 2 above)
Comment (robertkbk) [reply README.md:56]: I think I will make a small list of most useful commands.

Comment (t0ms0n00) [reply README.md:56]: migrate with --reset flag, imo it will be fine if we will have some kind of list

**Pull request merged**